src/include/distributed/metadata_cache.h

```diff
@@ -49,6 +49,7 @@ typedef struct
 
 	/* pg_dist_partition metadata for this table */
 	char *partitionKeyString;
+	Var *partitionColumn;
```

Comment:
I don't want to complicate things, but, do we ever need `partitionKeyString` after this change?

Reply:
Yes, it's still used in about 6 places.